### PR TITLE
DBZ-567 Adding example for MongoDB flattening SMT

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://issues.jboss.org/browse/DBZ-567

### CODE REVIEW

File: unwrap-mongodb-smt/README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Debezium Unwrap SMT Demo
+
+This setup is going to demonstrate how to receive events from MySQL database and stream them down to a PostgreSQL database and/or an Elasticsearch server using the [Debezium Event Flattening SMT](http://debezium.io/docs/configuration/event-flattening/).
```
Comment (jpechane) [inline unwrap-mongodb-smt/README.md]: Is there reference to Elasticsearch correct?

Comment (gunnarmorling) [reply unwrap-mongodb-smt/README.md]: No :) It's a left-over from the original unwrap example.

**Comment by gunnarmorling** [comment]
@jpechane pushed an update to fix the incorrect README.

**Comment by jpechane** [comment]
@gunnarmorling Merged, thanks!

**Pull request merged**